lib/rspecq/worker.rb

```diff
@@ -223,7 +222,13 @@ def files_to_example_ids(files)
         return files
       end
 
-      JSON.parse(out)["examples"].map { |e| e["id"] }
+      # strip extraneous output before json
+      rspec_json = out[out.index(/{.*version/)..-1]
```

Comment:
I'd prefer if we find a way to support arbitrary "split" commands (provided by the user), as opposed to doing things that are very specific to some use case.

I already dislike the fact that we do `DISABLE_SPRING=1 bundle exec rspec ...` since users may have other ways to execute rspec or may want to actually use spring, and this proposed change takes things even further towards the direction we'd like to avoid.

So I was thinking to have something like `Worker.slow_file_split_command=` that would accept a lambda provided by the user. The question is how this is going to be supported at the CLI level, since it'd require users to re-implement all the logic found in `bin/rspecq`. Maybe we could extract the whole CLI logic to a class that would be re-usable, so users would just do:

```
# myapp/bin/custom_rspecq
worker, reporter = RSpecQ::CLI.parse!
worker.file_splitter = ->() { ... }
worker.work
```

Reply:
I like this idea. Let's close this PR and I will open another.